activemodel/test/models/person.rb

```diff
@@ -9,6 +9,10 @@ class Person
   def condition_is_true
     true
   end
+
+  def condition_is_true_but_its_not
```

Comment:
I think we should rename this to `condition_is_false`

Reply:
ok will do

Reply:
renamed